airbyte-integrations/connectors/source-chargebee/source_chargebee/chargebee.yaml

```diff
@@ -0,0 +1,332 @@
+version: "0.1.0"
+
+definitions:
+  nested_selector:
+    type: RecordSelector
+    extractor:
+      type: CustomRecordExtractor
+      class_name: source_chargebee.components.NestedDpathExtractor
+      field_pointer: [ "list", '*', "{{ options.get('name') }}" ]
+  custom_field_transformation:
+    type: CustomTransformation
+    class_name: source_chargebee.components.CustomFieldTransformation
+  requester:
+    type: HttpRequester
+    url_base: "https://{{ config['site'] }}.chargebee.com/api/v2"
+    http_method: "GET"
+    authenticator:
+      type: BasicHttpAuthenticator
+      username: "{{ config['site_api_key'] }}"
+    error_handler:
+      type: CompositeErrorHandler
+      error_handlers:
+        - type: DefaultErrorHandler
+          response_filters:
+            - error_message_contains: "This API operation is not enabled for this site"
+              action: IGNORE
+              error_message: "Stream is available only for Product Catalog 1.0"
+        - type: DefaultErrorHandler
+          backoff_strategies:
+            - type: WaitTimeFromHeader
+              header: "Retry-After"
+  date_stream_slicer:
+    type: DatetimeStreamSlicer
+    start_datetime:
+      datetime: "{{ format_datetime(config['start_date'], '%s') }}"
+      datetime_format: "%s"
+    end_datetime:
+      datetime: "{{ now_utc().strftime('%s') }}"
+      datetime_format: "%s"
+    datetime_format: "%s"
+    cursor_granularity: PT1S
+    step: P100Y # set long interval to bypass slicing
```

Comment:
Chargebee API works weird, because it have an option to set both `updated_after` and `updated_before` parameters in request, but in result it returns all the records (logical `OR` instead of `AND`).
So I need to use only `updated_after` in request and then pagination.

Reply:
Chargebee API works weird, as if we pass both `updated_after` and `updated_before` parameters in request, it returns all records (logical `OR` instead of `AND` is used). Thay have special filter `between`, which should looks like `updated[between]=[start_datetime, end_datetime]`, but we have no option for this case in low-code.
Right now it is enough to use just updated_after and pagination

Reply:
@artem1205 you should be able to do this by removing the `start_time_option` and adding a request parameter to the requester
```
request_parameters:
 "updated[between]": "\"[{{stream_slice['start_time']}}, {{stream_slice['end_time']}}]\""
```

to produce a request like
` "url": "https://airbyte-test.chargebee.com/api/v2/subscriptions?sort_by%5Basc%5D=updated_at&include_deleted=true&updated%5Bbetween%5D=%5B1621695464%2C+1675133396%5D"`

Reply:
updated